FORGE/LUISSOARES/field_capacity.py

```diff
@@ -1,6 +1,13 @@
 import numpy as np
 
-def vol_water_content(C, S, OM, D, topsoil):       #C - Percentage of clay; S - Percentage of silt; OM - Percentage of organic matter; D - Bulk density; topsoil - qualitative variable
+def vol_water_content(C, S, OM, D, topsoil):
+    
+    """
+    
+    C - Percentage of clay; S - Percentage of silt; OM - Percentage of organic matter; D - Bulk density; topsoil - qualitative variable
```

Comment:
its a good try... :)
but try this:
install the extension autodocstring in VSCode and type """ again